pom.xml

```diff
@@ -41,7 +41,7 @@ The Initial Developer of the Covered Software is
     <tc-shader.version>2.0</tc-shader.version>
     <skip.deploy>false</skip.deploy>
     <!-- can't default to ${project.version}, breaks child projects like ehcache that use a different version -->
-    <thirdparty-bom.version>4.4.0-SNAPSHOT</thirdparty-bom.version>
+    <thirdparty-bom.version>4.4.1-SNAPSHOT</thirdparty-bom.version>
```

Comment:
I don't believe that there is one at the moment.  It's weird because aliasing was done in CI... they could share a BOM problably, a branch would be more clear.